backend/baton/src/test/java/touch/baton/assure/runnerpost/RunnerPostReadAssuredTest.java

```diff
@@ -91,4 +96,25 @@ class RunnerPostReadAssuredTest extends AssuredTestConfig {
                 .러너_게시글_생성_성공을_검증한다()
                 .생성한_러너_게시글의_식별자값을_반환한다();
     }
+
+    @Test
+    void 태그_검색에_성공한다() {
+        // given
+        tagRepository.save(TagFixture.create(new TagName("java")));
+        tagRepository.save(TagFixture.create(new TagName("javascript")));
+        tagRepository.save(TagFixture.create(new TagName("script")));
```

Comment:
픽스처 사용해주셔도 좋을거 같습니다~
```suggestion
        tagRepository.save(TagFixture.create(TagNameFixture.tagName("java")));
        tagRepository.save(TagFixture.create(tagName("javascript")));
        tagRepository.save(TagFixture.create(tagName("script")));
```